iggtools/subcommands/midas_run_snps.py

```diff
@@ -0,0 +1,328 @@
+import json
+import os
+from collections import defaultdict
+import multiprocessing
+
+import numpy as np
+from pysam import AlignmentFile  # pylint: disable=no-name-in-module
+import Bio.SeqIO
+
+from iggtools.common.argparser import add_subcommand
+from iggtools.common.utils import tsprint, num_physical_cores, command, InputStream, OutputStream, multithreading_hashmap, download_reference
+from iggtools.params import outputs
+from iggtools.models.uhgg import UHGG
+from iggtools.common.samples import parse_species_profile, select_species
+from iggtools.common.bowtie2 import build_bowtie2_db, bowtie2_align, samtools_index
+
+
+DEFAULT_ALN_COV = 0.75
+DEFAULT_SPECIES_COVERAGE = 3.0
+DEFAULT_ALN_MAPID = 94.0
+DEFAULT_ALN_MAPQ = 20
+DEFAULT_ALN_READQ = 20
+DEFAULT_ALN_BASEQ = 30
+DEFAULT_ALN_TRIM = 0
```

Comment:
I almost feel like it would be neat at some point to move these defaults into the bowtie2 module you just created, and change each one to be a dict, like so

DEFAULT_ALN_READQ = { "pangenome":  ...., "snps": .... }

May be even just parameterize the "add_argument" code in a shared func there.

Reply:
This comment isn't fully baked yet.